projects/self_taught_evaluator/README.md

```diff
@@ -0,0 +1,85 @@
+# Self-Taught Evaluators
+
+<p align="center"><img width="90%" src="figures/self_taught_dpo.png" /></p>
+
+Instructions and materials presented here correspond to the [Self-taught evaluators](https://arxiv.org/abs/2408.02666) research project.
```

Comment:
research project -> research paper?
this links to paper

Reply:
```suggestion
Instructions and materials presented here correspond to the [Self-taught evaluators](https://arxiv.org/abs/2408.02666) research paper.
```